src/quo2/components/community/community_link_card.cljs

```diff
@@ -0,0 +1,136 @@
+(ns quo2.components.community.community-card
+(:require [quo2.components.markdown.text :as text]
+          [quo2.foundations.colors :as colors]
+          [quo2.components.community.style :as style]
+          [quo2.components.icon :as icons]
+          [react-native.core :as rn]
+          [status-im.i18n.i18n :as i18n]))
+
+
+(defn community-title-and-description [title description image]
+  [rn/view
+   [rn/view
+    {:style {:padding-left    15
+             :flex-direction  :row
+             :align-items     :center}}
+
+    [rn/image
+     {:source image
+      :style {:width  16
+              :height 16}}]
+
+    [rn/view
+     {:style {:margin-left 5}}
+     [text/text {:font-size    15
+                 :font-weight  600
+                 :font-style   :normal
+                 :weight       :semi-bold}
+      title]]]
+
+   [rn/view
+    {:style {:padding-left 15}}
+    [text/text {:font-size       13
+                :font-style      :normal
+                :font-weight     400
+                :letter-spacing  -0.003
+                :weight          :regular}
+     description]]])
+
+(defn community-stats [{:keys [member_count active_members mutual_contacts]}]
+  [rn/view
+   {:style {:flex-direction    :row
+            :align-self        :flex-start
+            :margin-vertical   10
+            :width             271
+            :padding-left      15}}
+
+   [rn/view
+    {:style {:flex-direction  :row
+             :align-items     :center}}
+
+    [rn/view
+     [icons/icon  :i/add-user
+      {:size    20
+       :color   colors/neutral-50}]]
+
+    [text/text {:font-size    13
+                :font-weight  400
+                :font-style   :normal}
+     member_count]]
+
+   [rn/view
+    {:style {:flex-direction      :row
+             :align-items         :center
+             :padding-horizontal  10}}
+
+    [rn/view
+     [icons/icon  :i/add-user
+      {:size   20
+       :color  colors/neutral-50
+       :left   10}]]
+
+    [text/text {:font-size    13
+                :font-weight  400
+                :font-style   :normal}
+     active_members]]
+
+   [rn/view
+    {:style {:flex-direction  :row
+             :align-items     :center}}
+
+    [rn/view
+     [icons/icon :i/placeholder
+      {:size   20
+       :color  colors/neutral-50}]]
+
+    [text/text {:font-size    13
+                :font-weight  400
+                :font-style   :normal}
+     mutual_contacts]]])
+
+
+(defn community-link-card [{:keys [title description image member_count active_members mutual_contacts]}]
```

Comment:
we don't use `_` for variable names. Can you change these to `member-count`, `active-members` etc